docs/manifest.json

```diff
@@ -1175,6 +1175,12 @@
 		"markdown_source": "../packages/components/src/v-stack/README.md",
 		"parent": "components"
 	},
+	{
```

Comment:
We might have to remove this somehow from the docs.

Reply:
It can be listed here:
https://github.com/WordPress/gutenberg/blob/f444058588e2a94d52da0dce8bf6cf987ee643de/docs/tool/manifest.js#L11

This approach doesn't scale well though 😅

Reply:
I think adding the path to that "ignore" array should be enough for now.